src/chip/src/components/Chip.test.js

```diff
@@ -0,0 +1,49 @@
+import toJson from 'enzyme-to-json';
+import { mount } from 'enzyme';
+
+import Chip from './Chip';
+import ChipColors from './colors';
+import { hexToRgb } from '../../../utils/testing/ColorUtils';
+
+import * as Colors from '../../../colors';
+
+const { NEUTRAL } = Colors;
+
+describe('Chip', () => {
+  describe('snapshots', () => {
+
+    test('should match snapshot', () => {
+      const wrapper = mount(<Chip />);
+      expect(toJson(wrapper)).toMatchSnapshot();
+    });
+
+  });
+
+  test('default', () => {
+    const wrapper = mount(<Chip />);
+    const chip = wrapper.find('div');
+    expect(chip).toHaveLength(1);
+    expect(window.getComputedStyle(chip.getDOMNode())).toHaveProperty('color', hexToRgb(NEUTRAL.N600));
+    expect(window.getComputedStyle(chip.getDOMNode())).toHaveProperty('background', hexToRgb(NEUTRAL.N00));
```

Comment:
I haven't seen `window.getComputedStyle` and `getDOMNode()` before, interesting. How did you come across this? 

Also, just curious, does the following work?

```
const chip = wrapper.find(Chip);
expect(chip).toHaveStyleRule('color', NEUTRAL.N600);
expect(chip).toHaveStyleRule('background', NEUTRAL.N00);
```

Reply:
`const chip = wrapper.find(Chip);` works, but the following lines doo not. I found getComputedStyle when researching how to test css attributes of MUI components.